New version: CurvilinearGrids v0.3.0

JuliaRegistrator opened the pull request:
- Registering package: CurvilinearGrids
- Repository: https://github.com/smillerc/CurvilinearGrids.jl
- Created by: @smillerc
- Version: v0.3.0
- Commit: a796ddc52ba4c8ddeb895b9363afb5122850bee1
- Reviewed by: @smillerc
- Reference: https://github.com/smillerc/CurvilinearGrids.jl/commit/a796ddc52ba4c8ddeb895b9363afb5122850bee1#commitcomment-138445027
- Description: A Julia library for working with curvilinear grids
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
- Major rework of the conserved metric calculations for 3D
- Metrics are now members of the grid types, e.g. `grid.cell_center_metrics` for the normal metrics ($J, \xi_x, \eta_x, ...$) and `grid.edge_metrics` for the conserved metric terms $\hat\xi_x, \hat\eta_x$ at the edge $i+1/2$
`````
<!-- END RELEASE NOTES -->

<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd680ec5cd1e02750252dd7068f4b145f25c8d039ef492df0b5ec70376eae7e204378943aae080555fc3b7fcbbb64ad4439d725ab1a29603c3d570bf5e26a8b2d41879e948f8b34f028cae2e011ee83bdf47e9c5dd499a742597343356638e9fa5ac0e300ac2bccfb99353b1fc0a4736efce5eb7bbaaeb2a3f7743a4ee19af96535e162ff83665318859d268f9eb522c9f5d6f61d5bafaeb4dec22e58e4f704994315193f7255aaf94b7593ce683f518c12 -->

smillerc commented:
@JuliaRegistrator register

Release notes:

- Major rework of the conserved metric calculations for 3D
- Metrics are now members of the grid types, e.g. `grid.cell_center_metrics` for the normal metrics ($J, \xi_x, \eta_x, ...$) and `grid.edge_metrics` for the conserved metric terms $\hat\xi_x, \hat\eta_x$ at the edge $i+1/2$

JuliaRegistrator committed:
New version: CurvilinearGrids v0.3.0

UUID: 58b19789-dd19-4aa2-839c-c763a7b10301
Repo: https://github.com/smillerc/CurvilinearGrids.jl.git
Tree: a0a46e6c5c28b1de4435c20b0ec4c6b12108872b

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
